### ISSUE TITLE
rancher2_cluster - invalid or unknown key: kind

### ISSUE THREAD

**Issue by drpebcak** [rancher/terraform-provider-rancher2-archive#87]
When using the latest rc (v0.2.0-rc4), I receive this error when running terraform:
```
Error: module.example.rancher2_cluster.rke: : invalid or unknown key: kind
```

My `rancher2_cluster` resource looks like this:
```
resource "rancher2_cluster" "rke" {
  name        = "${var.vpc_name}"
  description = "TF created rke cluster"
  kind        = "rke"

  rke_config {
    cloud_provider {
      name = "aws"
    }
  }
}
```

This ran successfully on v0.2.0-rc3

**Comment by drpebcak** [comment]
Removing `kind` seems to work, so we should update the examples to remove that field.

**Comment by rawmind0** [comment]
Yeah, `kind` argument doesn't exist anymore on cluster definition. Docs are updated but i missed to remove it from examples. Submiting a PR to address it.

**Comment by rawmind0** [comment]
Addressed in PR #90

**Comment by drpebcak** [comment]
#90 resolves this!
